src/main/render/types.ts

```diff
@@ -104,7 +104,10 @@ export function typeNodeForFieldType(fieldType: FunctionType): ts.TypeNode {
       return createBooleanType()
 
     case SyntaxType.I64Keyword:
-      return ts.createTypeReferenceNode(COMMON_IDENTIFIERS.Int64, undefined)
+      return ts.createUnionTypeNode([
+        createNumberType(),
```

Comment:
Is there a reason we are doing this? It seems like we can always wrap an number in Int64 for fields of that type.

Reply:
My only reason is to make the `struct` constructor arguments containing `Int64` a bit more developer-friendly. I figured since the underlying library supports it and it was simple enough to implement, it was worth doing. Admittedly, I conflated the PR with this change.

Reply:
Wanted to give you a heads up we're planning to change this back. While it does clean up code for Clients it slightly complicates code for Service handlers everything needs to be wrapped in "if (typeof arg === 'number') { ... }" which isn't terrible, but given there's a trade off we want things to be type consistent. I was playing with having the generated code intelligently do type conversions based on what was most developer-friendly in a given context, but that was getting complicated and decided it wasn't terrible to make consumers wrap numbers in Int64 for that case.

Reply:
That makes sense. Thanks for the heads up!